### PR TITLE
Replace `Scheme` with a new repo

### PR DESCRIPTION
**Pull request by absop**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [ ] My package doesn't add key bindings. **
- [ ] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is [Scheme](https://github.com/absop/ST-Scheme)

My package is similar to [Scheme](https://github.com/egrachev/sublime-scheme) However it should still be added because it is better (supports more features) and still being maintained.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
In cases like these, if the original package isn't maintained (and you cannot be added as a maintainer, or transfer the repo, etc.), we can try to replace it. Would that work here (trying those other options first)?

**Comment by absop** [comment]
> In cases like these, if the original package isn't maintained (and you cannot be added as a maintainer, or transfer the repo, etc.), we can try to replace it. Would that work here (trying those other options first)?

Thanks for reply. I am contacting the author of the original package to replace it.

**Comment by egrachev** [comment]
Hello there! What I should do for you?

**Comment by braver** [comment]
There are many ways to go about this. If you guys agree that Schemer is a good improvement on Scheme, maybe do a PR and consider adding @absop as a maintainer?

**Comment by absop** [comment]
> There are many ways to go about this. If you guys agree that Schemer is a good improvement on Scheme, maybe do a PR and consider adding @absop as a maintainer?

Although they share the same feature of Syntax Highlighting, they are almost two completely different packages. I don't think do a PR is advisable. Maybe we should keep both of them and leave the choice to the users.

**Comment by braver** [comment]
We usually try to avoid that. To the end user they serve an identical purpose. It is not clear why you would pick one over the other. If there is a very good reason to keep an 8 years old package available we can do that, but for most users it will be beneficial to get the newest and most complete support for the language they’re using.

**Comment by absop** [comment]
If @egrachev agree, I would like to replace Scheme with Schemer. But let me give up [Schemer](https://github.com/absop/ST-Scheme) and to manage the [Scheme](https://github.com/egrachev/sublime-scheme), I can't do it.

**Comment by egrachev** [comment]
> If @egrachev agree, I would like to replace Scheme with Schemer. But let me give up [Schemer](https://github.com/absop/ST-Scheme) and to manage the [Scheme](https://github.com/egrachev/sublime-scheme), I can't do it.

I agree

File: repository/s.json [hunk 1]
```diff
@@ -414,12 +414,14 @@
 		},
 		{
 			"name": "Scheme",
-			"details": "https://github.com/egrachev/sublime-scheme",
-			"labels": ["language syntax", "snippets"],
+			"description": "Syntax Highlighting, smart snippets, code formatting, code navigation and more for the Scheme language",
```
Comment (braver) [inline repository/s.json:417]: You don’t have to put a description here, package control will take the description of the GitHub repo, allowing to change it at will.

Comment (absop) [reply repository/s.json]: Ok. Thank you for your remind.

**Comment by braver** [comment]
Alright, final thing would be to tag a release with your latest changes.

**Comment by absop** [comment]
> Alright, final thing would be to tag a release with your latest changes.

I've done it.
@braver

**Comment by egrachev** [comment]
Good

**Pull request merged**